### ISSUE TITLE
BAD_REQUEST, Missing partner_script_id parameter

### ISSUE THREAD

**Issue by ajith92** [Lenddo/node-lenddo#3]
I am following the [documentation](https://lenddo.com/documentation/nodejs_sdk.html), for getting the score. And I am getting the BAD REQUEST response.

**Code from documentation:**

``` javascript
var lenddo_clients = require('lenddo').clients;
var ScoreService = lenddo_clients.Score;
var client_instance = new ScoreService(id, secret);

client_instance.ClientScore.get(client_id).exec(function(err, result) {
  var response = result.response;
  console.log(response.data);
});
```

**Response I receive:**

``` javascript
{ request:
   { options:
      { host: 'scoreservice.lenddo.com',
        port: 443,
        path: '/ClientScore/<client_id>',
        method: 'GET',
        body: undefined,
        query: undefined,
        headers: [Object] },
     body: undefined,
     query: undefined },
  response:
   { code: 400,
     headers:
      { server: 'nginx/1.8.1',
        date: 'Wed, 19 Oct 2016 09:00:31 GMT',
        'content-type': 'application/json',
        'content-length': '75',
        connection: 'close' },
     raw: '{"message": "Missing partner_script_id parameter.", "name": "BAD_REQUEST"}\n',
     data:
      { message: 'Missing partner_script_id parameter.',
        name: 'BAD_REQUEST' } } }
```

**Comment by Howard3** [comment]
Hello Ajith,

Thank you for this detailed report. You're correct, we need to upgrade this SDK to support expanded requirements from our services. Please expect a published updated shortly. The syntax will change slightly to `client_instance.ClientScore.get(client_id, partner_script_id)` when it's complete.

Thank you again!
Best Regards,
Howard

**Comment by ajith92** [comment]
yes, Expected those syntax changes. Thank you.

**Comment by Howard3** [comment]
Hello @ajith92,

Thank you again for the report. I've now submitted a fix for this, v1.8.0 and I've tested it against your credentials and it was able to pull successfully!

Please pull from NPM and let me know if you have further issues. I'm going to close this ticket now.

Best Regards,
Howard
